posts/2024-03-04_inside_the__phar.../inside_the__pharmaverse.qmd

```diff
@@ -0,0 +1,136 @@
+Fortunately, some of you (you know who you are) innovated and created this fantastic channel for [Pharmaverse Blogs](https://pharmaverse.github.io/blog/).
+We can’t think of a better way to communicate than by the channel that the Pharmaverse community itself created.
+
+## Pharmaverse Inception
+
+The Pharmaverse Council was formed in 2020 as a coming together of 4 like-minded individuals embedded in the industry-wide efforts to increase the use of the R language for clinical trial analysis and reporting.
+These individuals included Ross Farrugia (Roche), Sumesh Kalappurakal (Johnson & Johnson), Michael Rimler (GSK), and Mike Stackhouse (Atorus).
+Each had historical experience with R adoption, through their primary roles within their respective organizations.
+
+Collectively, these 4 individuals saw the value in
+
+(i) reducing the choice set of R packages for users,
+
+(ii) identifying gaps in available R packages for delivering the clinical data pipeline, and
+
+(iii) encouraging the development of new packages and/or features to close those gaps.
+
+The result would be a subset of permissively licensed and open-sourced packages that anyone could use (or modify) to suit their specific use cases, thereby reducing the incidence of organizations privately solving typical problems in isolation.
+For more background, please refer to the [Pharmaverse Charter](https://pharmaverse.org/charter/).
+
+The current Pharmaverse Council companies include GSK, Roche, Johnson & Johnson, Atorus, and Novo Nordisk, represented by the original four founders and Ari Siggaard Knoph from Novo Nordisk.
+
+In March 2023, the [PHUSE](https://phuse.global/) Board of Directors approached the Pharmaverse Council with a proposal offering support to Pharmaverse developers and leadership to advance the Pharmaverse mission.
+The PHUSE proposal aligned with its mission to promote “\[s\]haring ideas, tools and standards around data, statistical and reporting technologies to advance the future of life sciences.” PHUSE is currently a supporter of the Pharmaverse and its objectives, evident through activities such as:
+
+-   the provision of access to PHUSE’s communication and file-sharing platforms to package teams,
+
+-   the provision of project management support to package teams (if needed),
+
+-   enablement and promotion of Pharmaverse innovations and activities, and
+
+-   enablement of face-to-face community connections at PHUSE events.
+
+## Pharmaverse Objective
+
+Pharmaverse aims to promote the collaborative development of curated open-source R packages for clinical reporting in pharma.
+Indirectly, we also strive to encourage and increase R adoption within the industry and facilitate communication and collaboration among both R users and R developers.
```

Comment:
Should we lead with this section?

Reply:
Good thought - I think it is okay as is.